point

```diff
@@ -87,17 +81,38 @@ program
 // program.option('--rpc <method> [params]', 'send a command to the daemon'); // todo
 program.parse(process.argv);
 
-// todo: mix other param arguments into config, like here:
-// let argv;
-// let config = rc('kadence', options(program.datadir), argv);
+const ctx = {};
 
-if (!program.config) {
-    program.config = path.join(program.datadir, 'config.json');
+if (program.datadir) {
+    process.env.DATADIR = program.datadir;
 }
 
-const ctx = {};
+function getContractAddress(name) {
+    const filename = path.resolve(__dirname, 'truffle', 'build', 'contracts', `${name}.json`);
+
+    if (!existsSync(filename)) {
+        return;
+    }
+
+    const {networks} = require(filename);
+
+    for (const network in networks) {
+        const {address} = networks[network];
+        if (address && typeof address === 'string') {
+            return address;
+        }
+    }
+}
+
+if (process.env.MODE === 'e2e') {
```

Comment:
I think for MODE we should have a file with all the available MODES in a constant

process.env.MODE === E2E

or

process.env.MODE === mode.E2E